Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -185,13 +189,16 @@ static void Read()
   // Reset rumble once on initial reading
   ResetRumble();
 
+  std::vector<u8> input_buffer;
```

Comment:
It looks like you add this as a `std::vector` in the first commit, but then change it to a `std::array` in the next commit. Is it possible to make it always be an `std::array`?

Reply:
I'll just squash the commits. :D

Reply:
More smaller commits would be nicer, but the way they're currently split doesn't seem to be that helpful so squashing it is fine.